upgrading/automated_upgrades.adoc

```diff
@@ -756,6 +756,25 @@ You can set the `openshift_logging_elasticsearch_replace_configmap` parameter to
 ConfigMap can cause the upgrade to fail. The default is set to `false`. For more information, see the parameter 
 in xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[specify logging Ansible variables].
 
+. Back up disks.
+
+. Review the link:https://www.elastic.co/guide/en/elasticsearch/reference/5.5/breaking-changes-5.0.html[Elasticsearch breaking changes].
+The breaking changes contains important information you need to be aware of when migrating your application to Elasticsearch 5.0.
+
+. Run the following command to list all fields in the cluster that have a dot character in the name, such as *foo.bar*.
+For more context see https://bugzilla.redhat.com/show_bug.cgi?id=1666141#c5
++
```

Comment:
@jcantrill We were recently asked by Eric Rich to not use JQ commands in the docs as RHEL 7 doesn't support JQ.
https://bugzilla.redhat.com/show_bug.cgi?id=1764726#c6

Reply:
We'll have to figure out the python equivalent

Reply:
What is recommended instead of `jq`? Python script?

Reply:
> What is recommended instead of `jq`? Python script?

yes.  piping through a python script. not fun

Reply:
@sferich888 Can you advise JQ alternative here?  I'm not found of figuring out python to do something that another tool already does well

Reply:
@jcantrill I agree, the request here is mainly for of command output parsing. Because your using exec to pull things out of ES the context of the original request doesn't quite apply/work here. 

Thus JQ is probably the best tool, for this (I just wish it was a dependency we could resolve). We just need to clearly call out that this command uses JQ, as an example to get directly at the necessary data and it's dependency can't be meet by RHEL. Please add a note or call out to this effect, and we can consider this a exception to the BZ request. 

I would also request that we not pipe to JQ multiple times and then pipe to grep/egrep. This overcomplicates things. 

Something like the following is better: 

`| jq ''.[].mappings[].properties | keys[] | select(., contains("\."))'` @jcantrill can we see if a command improvement in this area is possible? I think I got the syntax her correct but I'm not 100% sure, this my example may need some tweaking.

Reply:
@sferich888 Not sure what you mean by _dependency can't be meet by RHEL_.

Note:
The following command uses the *jq* JSON processor to get directly at the necessary data. RHEL cannot meet *jq*'s dependencies.

Reply:
Red Hat doesn't provide a `jq` package, for users to use. Thus, users are forced, if we use `jq` in our docs to find the command, install it and then use our docs. 

This is why were trying to phase out the use of the tool in our docs.

Reply:
@sferich888 Is this strong enough? 
```
"The following command uses the *jq* JSON processor to get directly at the necessary data. {product-title} does not provide a *jq* package. 
To use this command, you will need to download and install *jq*."
```

Reply:
Product title is RHEL, but yes this works. 
You might consider: 

{product-title}, depending on version, may not provide a package for `jq`. You may need to install this from external sources, or unsupported locations.